engines/python/setup/djl_python/scheduler/seq_batch_scheduler.py

```diff
@@ -87,8 +89,36 @@ def is_empty(self):
                    for seq_batcher_list in self.seq_batchers.values()
                    for seq_batcher in seq_batcher_list)
 
-    def inference_call(
-            self) -> Tuple[List[List[int]], List[int], List[List[int]]]:
+    def optimal_action(self,
+                       input_ids: torch.Tensor,
+                       request_uids: torch.Tensor,
```

Comment:
why request id in tensor format?

Reply:
The tensor format of request_id is used inside the scheduler, mainly to use the array indexing feature of the tensor. Here the request_id can be converted from List[int] to tensor outside of the add_request or optimal_action functions. But this conversion can be done inside these two functions as well if needed.